wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/settings/SettingsViewModel.kt

```diff
@@ -59,16 +59,24 @@ class SettingsViewModel @Inject constructor(
                     _state.update { it.copy(refreshState = refreshState) }
                 }
         }
+        viewModelScope.launch {
+            settings.warnOnMeteredNetwork.flow.collectLatest { warnOnMeteredNetwork ->
+                _state.update { it.copy(showDataWarning = warnOnMeteredNetwork) }
+            }
+        }
```

Comment:
This change was made so that the setting switch changes position automatically after refreshing.